cmd/kind/create/cluster/createcluster.go

```diff
@@ -26,15 +26,15 @@ import (
 	"sigs.k8s.io/kind/pkg/util"
 )
 
-type flags struct {
+type flagpole struct {
```

Comment:
this is unrelated but I realized I had `flags := &flags{}` when I was writing the new cobra commands so I'll soon change this in more places.